assets/crd/0000_20_topolvm.cybozu.com_logicalvolumes.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: logicalvolumes.topolvm.cybozu.com
```

Comment:
What's the current schedule for this API changing? We should make sure we have a jira ticket to figure out how to handle the transition within MicroShift.

Reply:
The API's groupVersionKind is expected to change in or by Q4 this year to replace the maintaining company's name.  There's not a set date for this yet.  General versioning has a spike already, but it'll be worth it to have this upcoming change tracked for awareness I think.

Reply:
I'll be very interested to hear what the ODF team's plan is for rolling upgrades with a GVK name change like that.

Reply:
I'm following up with them internally.  Upstream, there's a plan to implement a migration tool, but I don't know what plans for downstream users will be.

Reply:
May be a web hook in the meanwhile to provide translation between both versions?